.github/workflows/build-android.yml

```diff
@@ -0,0 +1,50 @@
+name: Test Android Pull Requests
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'android/**'
```

Comment:
You should probably also run this on `push` on main, with same path filter in it.

Reply:
Done, thanks :)